windows/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ set(SOURCES
     hidapi_hidsdi.h
 )
 
+if(HIDAPI_ENABLE_ASAN)
+    # Option /MTd needed to embedd ASAN library
+    add_compile_options(/MTd)
+endif()
```

Comment:
BTW: in my other project(s) I didn't it. for ASAN

Reply:
Static linking is not needed in general, you could als add the ASAN DLL, that comes in various flavors with VisualStudio, to your DLL search path.
But it's placed in a directory with a name like this:
`C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\bin\Hostx64\x86\clang_rt.asan_dbg_dynamic-i386.dll`
I'm not able to determine this path using CMake. It depends on many things like VisualStudio installation path, Visual Studio version and the build type.

Reply:
It is rather easy to get this path:
- it is `MSVC`-specitic (so unser `if(MSVC)`);
- as per [documentation](https://learn.microsoft.com/en-us/cpp/sanitizers/asan?view=msvc-170) - on MSVC the support is limited to x86 and x64 on Windows 10, so only one of two possible suffixes: `i386` or `x86_64`;
- the location is [documented](https://learn.microsoft.com/en-us/cpp/sanitizers/asan-known-issues?view=msvc-170#addresssanitizer-runtime-dll-locations) and it matches the path to `CMAKE_C_COMPILER` directory;

---

But in any case - I don't believe we need to handle that in our CMake. ASAN runs are designed to be executed only in development builds (e.g. no "production" scenarios).
The runtime is always available from MSVC development sonsole, so I believe that is just enough.